plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -180,6 +185,14 @@ protected Transaction newCreateTableTransaction(
         return createTableTransaction(schemaTableName.toString(), ops, metadata);
     }
 
+    protected abstract Optional<String> getOwner(ConnectorSession session);
+
+    @Override
+    public String latestMetadataLocation(ConnectorSession session, String tableLocation)
```

Comment:
I am not very sure about this change. If it makes sense to add this method to the abstract catalog. Please suggest.
CC: @alexjo2144 | @findinpath | @findepi